### DISCUSSION TITLE
Feature Request: Playback History and Autobookmarks

### DISCUSSION THREAD

**Discussion by GatorChomp21** [PaulWoitaschek/Voice#1420] [Ideas]
A menu where my position would be saved upon certain actions, ie Playback device changed, Sleep Timer started, Sleep timer ended, Play paused, Play resumed, playback speed changed. This could be kept in the bookmarks tab with different pages, one with manual bookmarks and the other with these autobookmarks.

**Comment by nikoss** [comment]
I thought of the same. Here is what I had prepared...
------
history (auto bookmarks, log)

So, what happens is that sometimes you accidentaly change the audio position.

You might be listening to an audio book and, when moving your phone around, move back or forward to the timeline.

Or you might change chapter.

Or try to start an audiobook and instead start a different one.

Or you might have set the timer wrong, and the audiobook played for hours, while you were asleep.

Etc, etc. 

And we are talking about trying to find your way back among hours and hours of audio.

A possible solution to this problem would be a history column. 

There you would find which audiobooks you played last; when started; when paused; when skipped; from when to when. 
By clicking on them, you would go back to that position.

(Should "Play" start automatically, or should you go to that point, but in a "Pause" condition? Maybe that could be an option?)

The history might look like this:

___
| "Lecture about something" |
|   00:00:00 play           |
___
| "Lecture about something" |
|   00:00:10 pause          |
___
| "Lecture about something"  |
|   00:00:10 skip, departure |
___
| "Lecture about something" |
|   00:05:13 skip, arrival  |
___
| "Lecture about something" |
|   00:05:13 play           |
___
| "Lecture about something" |
|   00:25:28 stop           |
___
| "History book about some issue, |
| having a very long title, that  |
| needs many lines to unfold      |
| itself"                         |
|  "Chapter 11: the great change  |
|  that would come"               |
|    01:36:52 play                |
___

Having a history or not should be optional (a privacy issue).

How many items listed in the history should also be an option. Someone might want 12 items. Another person might want 112 items.
